envoyauth/response.go

```diff
@@ -4,9 +4,6 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
-	"net/http"
-	"slices"
```

Comment:
I kind of appreciated the blocks here -- stdlib vs third-party. but that's not a blocker.